test/single_file/add.expected

```diff
@@ -1,21 +1,22 @@
+#include <cinttypes>
 #include <cstdlib>
 #include <functional>
 
 static bool __dredd_enabled_mutation(int local_mutation_id) {
   static bool initialized = false;
-  static bool enabled[13];
+  static uint64_t enabled_bitset[1];
   if (!initialized) {
     const char* __dredd_environment_variable = std::getenv("DREDD_ENABLED_MUTATION");
     if (__dredd_environment_variable != nullptr) {
       int value = atoi(__dredd_environment_variable);
       int local_value = value - 0;
       if (local_value >= 0 && local_value < 13) {
-        enabled[local_value] = true;
+        enabled_bitset[local_value / 64] |= (1 << (local_value & 63));
```

Comment:
It appears this should be % not &.

Reply:
Ah - I had forgotten to re-generate the expectation files. I initially planned to do `& 63`, which is the same but more efficient than `% 64`, but the compiler will do that optimisation and the latter is more reasonable.